client/layout/hosting-dashboard/item-view/index.tsx

```diff
@@ -64,11 +65,26 @@ export default function ItemView( {
 	// so it can't be used here to determine if the site is a staging site
 	const isStagingSite = itemData.subtitle?.toString().startsWith( 'staging-' );
 
-	const isAtomicSite = useSelector( ( state ) =>
-		isSiteWpcomAtomic( state, itemData.blogId ?? null )
-	);
+	const { data: atomicTransfer } = useQuery( {
+		...siteLatestAtomicTransferQuery( itemData.blogId ?? 0 ),
+		refetchInterval: ( query ) => {
+			return isAtomicTransferInProgress( query.state.data?.status ?? 'pending' ) ? 5000 : false;
+		},
+		enabled: !! itemData.blogId && isStagingSite,
+	} );
+
+	const { data: stagingSite } = useQuery( {
+		...siteByIdQuery( itemData.blogId ?? 0 ),
+		refetchInterval: 5000,
```

Comment:
I think we could consider stopping the refetch here as well - once we get the `jetpack_connection`.